process/sync/baseSync.go

```diff
@@ -29,6 +29,22 @@ const sleepTime = time.Duration(5 * time.Millisecond)
 // block through recovery mechanism, if its block request is not resolved and no new block header is received meantime
 const maxRoundsToWait = 5
 
+type notarizedInfo struct {
+	lastNotarized           map[uint32]uint64
+	finalNotarized          map[uint32]uint64
+	nonceWithLastNotarized  map[uint32]uint64
+	nonceWithFinalNotarized map[uint32]uint64
+	startNonce              uint64
+}
+
+func (ni *notarizedInfo) reset() {
+	ni.lastNotarized = make(map[uint32]uint64, 0)
+	ni.finalNotarized = make(map[uint32]uint64, 0)
+	ni.nonceWithLastNotarized = make(map[uint32]uint64, 0)
```

Comment:
suggestion to rename nonceOfLastNotarized

Reply:
Changed